samples/micrometer-samples-core/src/main/java/io/micrometer/core/samples/utils/SampleConfig.java

```diff
@@ -22,6 +22,6 @@
 public class SampleConfig {
     public static MeterRegistry myMonitoringSystem() {
         // Pick a monitoring system here to use in your samples.
-        return SampleRegistries.jmx();
+        return SampleRegistries.azure("");
```

Comment:
looks like some debug changes creeped to the PR

Reply:
Updated, thanks